### PR TITLE
Feat/edit saved class

### PR DESCRIPTION
**Pull request by adrianammt**
Hello all! Here is a new PR:

Main changes:
- new icon in PlayClassCard that controls the Edit function. On Edit, link to create page with preloaded class to edit. ( I think it would be a good idea here to add a cancel button in case, the user changes her/his mind).
- background is now an svg with subtle changes of color
- the create form shows a message if the classDuration is set to 00:00 
- fixed a bug in the handlePlayClick function that was affecting the resume of prepTimer

Looking forward to your feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## MYITA
+
+Meditation and Yoga interval timer App (MYITA) is a an interval timer App.
+
+Its's main use is for Meditation and Yoga, but it can be used for any activity that requires an interval timer with different alerts.
+
+You can save your classes/practices settings and have them ready to use. And those that you use the most can be saved in your favourites for easy access.
+
```
Comment (hellodebs) [inline README.md:8]: uuuuh, nice! I like the explanation in the readme file!

File: src/App.js [hunk 2]
```diff
@@ -41,17 +44,42 @@ function App() {
...
+    const index = listOfClasses.findIndex((savedClass) => savedClass.id === id);
+    setListOfClasses([
+      ...listOfClasses.slice(0, index),
+      updatedEntry,
+      ...listOfClasses.slice(index + 1),
+    ]);
+    toggleIsEdit();
+    toastSavedMessage("Class updated!");
+  }
+
```
Comment (hellodebs) [inline src/App.js:66]: good job!  Nice and clean and works perfectly!

File: src/components/PlayClassCard/PlayClassCard.js [hunk 3]
```diff
@@ -207,27 +215,38 @@ export default function PlayClassCard({ classToPlay, toggleFavourite }) {
...
-      </div>
-      <h3>Class</h3>
+      <h3>Preparation Time: {prepTime} sec</h3>
+
+      <p className="PrepTime-line__time--text">{prepFormattedTime}</p>
+      <div className="PrepTime-line"></div>
+      <div className="PrepTime-line__time"></div>
+
+      <h3>
+        Class:{" "}
```
Comment (hellodebs) [inline src/components/PlayClassCard/PlayClassCard.js:225]: The prettier extension should stop doing that :D

**Review by hellodebs — COMMENTED** [review]
Well done, Adriana! :) I got nothing make your code any better. 🥇

File: src/components/PlayClassCard/PlayClassCard.js (hunk 3 above)
Comment (adrianammt) [reply src/components/PlayClassCard/PlayClassCard.js:225]: Thanks for catching that. I actually have the prettier installed. Will take a look.

**Pull request merged**